docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+services:
```

Comment:
The Docker Compose version is set to `1`, which is outdated. Updating it to a more recent version, like `3.8`, will allow you to utilize newer features and improvements.